File: src/main/java/com/p2p/lending/dao/InvestInfoDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import org.apache.ibatis.annotations.Param;
import com.p2p.lending.entity.InvestInfo;

public interface InvestInfoDao extends BaseDao<Object,InvestInfo>{
	public List<InvestInfo> recordList(Map<String, Object> map);
	Double getSum(Map<String, Object> map);
	List<InvestInfo> getDtail(@Param("params") Map<String, InvestInfo> params);
	Integer getMoney(Integer uid);
}

File: src/main/java/com/p2p/lending/dao/InvestInfoDao.kt
package com.p2p.lending.dao

import org.apache.ibatis.annotations.Param
import com.p2p.lending.entity.InvestInfo

interface InvestInfoDao : BaseDao<Any, InvestInfo> {

    fun recordList(map: Map<String, Any>): List<InvestInfo>

    fun getSum(map: Map<String, Any>): Double?

    fun getDtail(@Param("params") params: Map<String, InvestInfo>): List<InvestInfo>

    fun getMoney(uid: Int?): Int?
}
